### ISSUE TITLE
rewrite default http config :client_max_body_size

### ISSUE THREAD

**Issue by Lekinio** [NginxProxyManager/nginx-proxy-manager#353]
trying to change client_max_body_size size by creatint file in /data/nginx/custom/http.conf 

but its return error duplicated directive

**Comment by nightcomdev** [comment]
I'm not sure but did you try just add it in "Advanced" when you edit domain? I mean this should resolve problem right? That's what this field is for I guess. for example:
`client_max_body_size 40m;`

**Comment by jc21** [comment]
Correct, because it's already [defined here](https://github.com/jc21/nginx-proxy-manager/blob/master/docker/rootfs/etc/nginx/nginx.conf)

`client_max_body_size          2000m;`

Is 2000mb a problem?

**Comment by Lekinio** [comment]
> defined here

yes and i want to edit it. i want to increase
